packages/components/src/select-control/types.ts

```diff
@@ -92,6 +90,11 @@ export type SelectControlMultipleSelectionProps = SelectControlBaseProps & {
 	 * @default false
 	 */
 	multiple: true;
+	/**
+	 * The value of the selected option.
+	 *
+	 * If `multiple` is true, the `value` should be an array with the values of the selected options.
+	 */
```

Comment:
As above: Does it make sense to make this more multiple-specific, so it's clear that it's always an array of value**s** and not a single value since it's only when the select field is a multiple select one?